### PR TITLE
Improve Iceberg documentation for refresh new materialized view

### PR DESCRIPTION
**Pull request by tlblessing**
## Description

Added mention as requested of the requirement to manually run REFRESH MV after creating it.

## Release notes

(x) This is not user-visible or docs only and no release notes are required.
( ) Release notes are required, please propose a release note for me.
( ) Release notes are required, with the following suggested text:

### CODE REVIEW

File: docs/src/main/sphinx/connector/iceberg.rst [hunk 1]
```diff
@@ -1385,6 +1385,10 @@ view definition. The ``iceberg.materialized-views.storage-schema`` catalog
 configuration property or ``storage_schema`` materialized view property can be
 used to specify the schema where the storage table is created.
 
+Creating a materialized view does not automatically populate it with data. After
+creating the materialized view, you must run :doc:`REFRESH MATERIALIZED VIEW
```
Comment (mosabua) [inline docs/src/main/sphinx/connector/iceberg.rst:1389]: I dont think you need to duplicate the title in the doc link (see below)

File: docs/src/main/sphinx/connector/iceberg.rst [hunk 2]
```diff
@@ -1385,6 +1385,10 @@ view definition. The ``iceberg.materialized-views.storage-schema`` catalog
...
+</sql/refresh-materialized-view>` to populate data in the materialized view.
+
 Updating the data in the materialized view with
 :doc:`/sql/refresh-materialized-view` deletes the data from the storage table,
```
Comment (mosabua) [inline docs/src/main/sphinx/connector/iceberg.rst:1393]: Since we link above .. we can just use code syntax here now

**Review by mosabua — CHANGES_REQUESTED** [review]

File: docs/src/main/sphinx/connector/iceberg.rst [hunk 3]
```diff
@@ -1385,22 +1385,25 @@ view definition. The ``iceberg.materialized-views.storage-schema`` catalog
...
-view is queried, the snapshot-ids are used to check if the data in the storage
-table is up to date. If the data is outdated, the materialized view behaves
-like a normal view, and the data is queried directly from the base tables.
-Detecting outdated data is possible only when the materialized view uses
-Iceberg tables only, or when it uses mix of Iceberg and non-Iceberg tables
-but some Iceberg tables are outdated. When the materialized view is based
-on non-Iceberg tables, querying it can return outdated data, since the connector
-has no information whether the underlying non-Iceberg tables have changed.
+Creating a materialized view does not automatically populate it with data. You
+must run :doc:`REFRESH MATERIALIZED VIEW </sql/refresh-materialized-view>` to
```
Comment (mosabua) [inline docs/src/main/sphinx/connector/iceberg.rst:1389]:
```suggestion
must run :doc:`/sql/refresh-materialized-view` to
```

**Review by mosabua — CHANGES_REQUESTED** [review]

**Comment by mosabua** [comment]
Also fix spelling in commit message please @tlblessing

File: docs/src/main/sphinx/connector/iceberg.rst [hunk 4]
```diff
@@ -1385,22 +1385,25 @@ view definition. The ``iceberg.materialized-views.storage-schema`` catalog
...
+populate data in the materialized view.
+
+Updating the data in the materialized view with ``REFRESH MATERIALIZED VIEW``
```
Comment (mosabua) [inline docs/src/main/sphinx/connector/iceberg.rst:1392]: This is just a link removal and not content changes right?

Comment (tlblessing) [reply docs/src/main/sphinx/connector/iceberg.rst:1392]: right

**Review by mosabua — APPROVED** [review]

**Pull request merged**